tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,485 @@
+"""
+Provide tests for command line interface's get list of batches command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_list_batches_with_all_parameters(mocker):
+    """
+    Case: get a list of batches by identifiers, identifier starting from, limit, head, reverse.
+    Expect: batches are returned from a node on localhost.
+    """
+    batch_ids = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+                '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751, ' \
+                '257353cb1180bdce4e19f290e4bfdb48212744a8ae08f6fc974e8c7c2cfe4692' \
+                '19d257a7c62197bd614b9dc13cd562c6612aee45ab0bccfcdd6b0d1ed0b3cdcf'
+
+    start = '257353cb1180bdce4e19f290e4bfdb48212744a8ae08f6fc974e8c7c2cfe4692' \
+            '19d257a7c62197bd614b9dc13cd562c6612aee45ab0bccfcdd6b0d1ed0b3cdcf'
+
+    head = '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+           '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--ids',
+        batch_ids,
+        '--start',
+        start,
+        '--limit',
+        1,
+        '--head',
+        head,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+        '--reverse',
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_with_ids():
+    """
+    Case: get a list of batches by identifiers.
+    Expect: batches with specified identifiers are returned.
+    """
+    batch_ids = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+                '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751, ' \
+                '257353cb1180bdce4e19f290e4bfdb48212744a8ae08f6fc974e8c7c2cfe4692' \
+                '19d257a7c62197bd614b9dc13cd562c6612aee45ab0bccfcdd6b0d1ed0b3cdcf'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--ids',
+        batch_ids,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_header_signature = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+                                '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751'
+
+    result_header_signature = json.loads(result.output).get('result')[0].get('header_signature')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert expected_header_signature == result_header_signature
+
+
+def test_get_list_batches_with_non_existing_ids():
```

Comment:
Make one test case using `@pytest.mark.parametrize` for all tests with non-existing arguments (ids, start, head), they have the same error message `'List of batches not found.'`.

Reply:
Done.